src/util/cmdlineargs.cpp

```diff
@@ -52,12 +53,23 @@ bool CmdlineArgs::Parse(int &argc, char **argv) {
         } else if (argv[i] == QString("--timelinePath") && i+1 < argc) {
             m_timelinePath = QString::fromLocal8Bit(argv[i+1]);
             i++;
+        } else if (argv[i] == QString("--debugLevel") && i+1 < argc) { // Set cmdline message verbosity
+            bool isInt; // Scoped variable, will be destroyed at the end of this block @vhexs
+            m_debugLevel = QString::fromLocal8Bit(argv[i+1]).toInt(&isInt); // Try conversion, if it's not an int
+                                                                            // then the if statement below will
+                                                                            // set m_debugLevel back to its
+                                                                            // default value of 1
+            if (!isInt) {
+                fputs("\ndebugLevel argument wasn't a number! Mixxx will only output\n\
+warnings and errors to the console unless this is set properly.\n", stdout);
+                m_debugLevel = 1;
```

Comment:
Ah, I see what you mean now. I'll try using integers as I'm not very good with enums.